packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,68 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+## PatternFly 5.3
+
+PatternFly 5.3 is now live! Here are the highlights from this release. 
+
+### React templates 
+We created a new feature for React components, called "templates", which combine a component with logic that supports a specific use case. Templates also come with a streamlined API that offers additional, limited customization. This will help reduce the amount of boilerplate code needed for common and recurring use cases.
+
+Templates will also make it easier to upgrade your deprecated components going forward. For example, you can use a template to upgrade from the [current version of our select component](/components/menus/select) from [the deprecated version.](/components/menus/select/react-deprecated)
+
+### Component updates 
+
```

Comment:
We also added the Next dual list selector component

Reply:
@tlabaj I added this in!